src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourceMappingOptionsControl.xaml

```diff
@@ -0,0 +1,56 @@
+<UserControl x:Class="NuGet.Options.PackageSourceMappingOptionsControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             Background="{DynamicResource {x:Static SystemColors.ControlBrushKey}}"
+             Foreground="{DynamicResource {x:Static SystemColors.ControlTextBrushKey}}"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+  <Grid Margin="12">
+    <Grid.RowDefinitions>
+      <RowDefinition Height="24"/>
+      <RowDefinition Height="144*"/>
+      <RowDefinition Height="223*"/>
+      <RowDefinition Height="36"/>
+    </Grid.RowDefinitions>
+    <Grid.Resources>
+      <!-- Data Template for list items -->
+      <DataTemplate x:Uid="MappingItemTemplate" x:Key="MappingItemTemplate">
+        <StackPanel x:Uid="ListItemStackPanel" Margin="0,3,0,3">
+          <TextBlock FontWeight="Bold">
+            <TextBlock.Text>
+              <Binding Path="ID"/>
+            </TextBlock.Text>
+          </TextBlock>
+          <TextBlock>
+            <TextBlock.Text>
+              <Binding Path="SourcesString"/>
+            </TextBlock.Text>
+          </TextBlock>
+        </StackPanel>
+      </DataTemplate>
+    </Grid.Resources>
+    <TextBlock  Grid.Row="0" Margin="6,0,0,0" Text="{x:Static nuget:Resources.VSOptions_Label_PackageSourceMappings}"/>
+    <ListBox Name="packageList"
```

Comment:
This seems like a WPF bug: some selection is not completely filled when you scroll horizontally:

![image](https://user-images.githubusercontent.com/1192347/183227808-868b0d37-8908-4549-9a2e-3ec53c0e9bd9.png)

Reply:
@dominoFire I was unable to repro this. Do you recall if you were using particular scaling/resolution? It may be the pseudo-localization making a panel grow larger than normal. If you agree, I'll move this to th